### PR TITLE
PCB -> ACC

### PR DESCRIPTION
**Pull request by discip**
@Ralim
Please do not get angry, since I do not know much about the internal linkages, I am just trying.
Since you stated that the naming is not ideal, I tried to name it appropriately. 😁

If I am barking up the wrong tree, please let me know. Also please tell me how this should be done correctly, if you don't mind.

### CODE REVIEW

**Comment by Ralim** [comment]
Seems sane so far.
Though I would use a scoped enum for the values

So add to a header file something like:

```
enum class AccelType {
    MMA=1,
    LIS=2,
    .....
    None=99,
    
```

And use it like `AccelType::MMA`

Why?
Because it means the compiler type checks it :) but you can always cast them back to int's as well

**Comment by discip** [comment]
@Ralim
First of all thank you, for helping me. ❤
I really do not know, if I am doing the right thing. 😵
I am not even  able to decipher the error message. Sigh. 😩😭🤦‍♂️

**Comment by Ralim** [comment]
You are on the right path

` error: no match for 'operator==' (operand types are 'uint8_t' {aka 'unsigned char'} and 'AccelType')`
This is a very roundabout way of sating the type on the left and right of the `==` are not _directly_ comparable.
Then its expanding to say the left is a `uint8_t` and the right is an `AccelType`
(And now the graphic should make more sense too)

The easiest would be to change the type of the thing on the left `DetectedAccelerometerVersion` to be declared as `AccelType` instead.

Or you could cast the `AccelType` to an int to compare it `(int)AccelType::LIS`

**Comment by discip** [comment]
@Ralim
The last try kind of worked, because it still shows the value e.g. **`ACC 2`** instead **`ACC LIS`**. 😵

**Comment by Ralim** [comment]
Thats good :)

So the hardest part will be getting the text instead of the number.
As you will need to:
- Add the text to the translation python code; as the code for the letters varies by language
- Then lookup the correct one of those strings by the number from the enum
Easist is probably to follow the paths that make the debug menu lookup for the text on the left of the screen, and copy that to create a new array of "names"

**Comment by Ralim** [comment]
BTW, if you get fed up I can take this on, but also prefer to help you gain experience if you wish; just let me know if you get "over" it

**Comment by discip** [comment]
@Ralim
I would really love to get this done, but since I am lacking the understanding of the dependencies and the terms you are using, I think it would be best, if you would take it on this time, with me watching every single step you are doing. And hopefully I will be able to get it along the way. I hope you don't mind me asking, why you did this or that, whilst checking what changes you made. 😃

**Comment by Ralim** [comment]
@discip 

No issue at all, just always like to let people try first before I jump on :)

Doing small commits with hopefully more helpful names :)

**Comment by Ralim** [comment]
I think this should be good now, @discip Can you give it a test?

**Comment by discip** [comment]
# Exceptional! 🤯👍

I clearly miscalculated regarding the effort required for this tiny bit of change! 🤯
And at the same time I am blown away by the speed and precision you implemented this with! 😲

**Comment by Ralim** [comment]
Eh the catch to firmware is that tiny changes explode into lots of bits and pieces. Thankfully the pattern was there to copy the debug menu and then its just a game of fixing compiler errors :)

File: source/Core/Threads/GUIThread.cpp [hunk 1]
```diff
@@ -713,7 +713,7 @@ void showDebugMenu(void) {
       break;
     case 10:
       // Print PCB ID number
```
Comment (discip) [inline source/Core/Threads/GUIThread.cpp:715]: This should read:
// Print ACC type
Am I right? 😁

Comment (Ralim) [reply source/Core/Threads/GUIThread.cpp:715]: Eugh yeah
missed that 🙃

Comment (discip) [reply source/Core/Threads/GUIThread.cpp:715]: Sorry was a bit too late. 😅

**Pull request merged**